fuzzysort.d.ts

```diff
@@ -0,0 +1,129 @@
+/**
+ * This is patch of the original type definition included with the module, because it is lying!
+ * The original has highlight(result?: Result, ...) but actually it is highlight(result: Result | null)!
+ */
```

Comment:
We only need to change one method definition, but sadly there doesn't seem to be a way to do that "more nicely" with TS.
Also, I don't like this file being in the project root - maybe we should create a folder `types` and put all type overrides there?

Reply:
1. Could we use an "any" cast or some wrapper or other smaller hack in our code with a comment explaining the issue, instead of copying in the full d.ts?
2. In addition submit an upstream patch with your fix? Sometimes you get lucky and get a fast turnaround time.

Reply:
1. Probably an `as Result` would be enough, but we have similar code in multiple places and I would have to explain that in all those places. Can look into writing a wrapper function though, so we only call one function.
2. The repo looks pretty much dead, sadly. https://github.com/farzher/fuzzysort